tests/agents/test_opencode_mcp_integration.py

```
@@ -0,0 +1,397 @@
+        mcp_config = {
+            "mcp": {
+                "codeboarding": {
+                    "type": "local",
+                    "command": ["python", "-m", "codeboarding_mcp_server"],
+                }
+            }
+        }
+        env = os.environ.copy()
+        env["OPENCODE_CONFIG_CONTENT"] = json.dumps(mcp_config)
+
+        subprocess.Popen(["opencode", "serve", "--port", "4096"], env=env)
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)